runtime/makelib/targets.mk.linux.inc.ftl

```diff
@@ -48,7 +48,6 @@ $(UMA_DLLTARGET) : $(UMA_OBJECTS) $(UMA_TARGET_LIBRARIES)
 		$(UMA_DLL_LINK_POSTFLAGS)
 ifdef j9vm_uma_gnuDebugSymbols
 	$(OBJCOPY) --only-keep-debug $@ $(@:$(UMA_DOT_DLL)=.debuginfo)
-	$(OBJCOPY) --strip-debug $@
```

Comment:
Please undo changes to this file.

Reply:
Thanks for catching this! It got removed by a script given for helping with debugging but wasn't meant to be commited.